PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azuretools/ijidea/ui/DeviceLoginWindow.java

```diff
@@ -102,6 +110,7 @@ private void pullAuthenticationResult(final AuthenticationContext ctx, final Dev
                 }
             }
         }
+        authLogger.setLevel(authLoggerLevel);
```

Comment:
It might be better to put this line to a try-finally closure, the above `catch` doesn't catch all exceptions, the uncaught exceptions may cause the logger level not restored.

Reply:
done